thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiLog.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2016 grandcentrix GmbH
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.grandcentrix.thirtyinch;
+
+
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+/**
+ * Logging class used for all logging of ThirtyInch.
+ */
+public class TiLog {
+
+    /**
+     * Simple logging interface for log messages from ThirtyInch
+     *
+     * @see #setLogger(Logger)
+     */
+    public interface Logger {
+
+        /**
+         * @param level one of {@link Log#VERBOSE}, {@link Log#DEBUG},{@link Log#INFO},
+         *              {@link Log#WARN},{@link Log#ERROR}
+         * @param tag   log tag, caller
+         * @param msg   message to log
+         */
+        void log(final int level, final String tag, final String msg);
+    }
+
+    private static final String TAG = "ThirtyInch";
+
+    // logs everything to Timber by default
```

Comment:
This is wrong since https://github.com/grandcentrix/ThirtyInch/pull/14/commits/4da934b2989a0fbc99120cc421677641cc87a21b